tests/plugins/link/link.js

```diff
@@ -592,7 +592,53 @@
 			correctInput: '123456789',
 			incorrectLinkAssertionCallback: assertIncorrectLinks,
 			correctLinkAssertionCallback: assertCorrectLinks
-		} )
+		} ),
+
+		// (#2478)
+		'test Ctrl+K keystroke': function() {
+			var bot = this.editorBots.noValidation,
+				editor = bot.editor;
+
+			bot.setData( '', function() {
+				editor.once( 'dialogShow', function( evt ) {
+					resume( function() {
+						var dialog = evt.data;
+
+						dialog.hide();
+						assert.areSame( 'link', dialog._.name );
+					} );
+				} );
+
+				editor.editable().fire( 'keydown', new CKEDITOR.dom.event( {
+					keyCode: 75,
+					ctrlKey: true
+				} ) );
+				wait();
+			} );
+		},
+
+		// (#2478)
```

Comment:
This case is not a part of the issue so I would remove tag reference here.

Reply:
I'd keep the reference and treat this one as prevention from regression (accidentally breaking the former keystroke).

Reply:
I'd second to what @jacekbogdanski noted, this case is not related. Regression protection (related to #2478) is provided by the `test Ctrl+K keystroke` test case.